includes/Modules/Analytics.php

```diff
@@ -325,6 +325,23 @@ protected function print_amp_gtag() {
 			),
 		);
 
+		/**
+		 * Filters configuration options for the object that is passed to amp-analytics as a JSON component.
```

Comment:
This should clarify it's using gtag, similar to the non-AMP filter: `Filters the gtag configuration options for the amp-analytics tag.`